hotspot/make/bsd/makefiles/gcc.make

```diff
@@ -145,11 +145,21 @@ ifeq ($(USE_CLANG), true)
     # level specific PCH files for the opt build and use them here, but
     # it's probably not worth the effort as long as only a few files
     # need this special handling.
-    PCH_FLAG/loopTransform.o = $(PCH_FLAG/NO_PCH)
     PCH_FLAG/sharedRuntimeTrig.o = $(PCH_FLAG/NO_PCH)
     PCH_FLAG/sharedRuntimeTrans.o = $(PCH_FLAG/NO_PCH)
-    PCH_FLAG/unsafe.o = $(PCH_FLAG/NO_PCH)
-  
+
+    ifeq ($(shell expr $(CC_VER_MAJOR) = 4 \& $(CC_VER_MINOR) = 2), 1)
```

Comment:
These look like gcc version numbers?  I'm asking because we're in a USE_CLANG block, so we shouldn't be caring about gcc version.

This might be because you're trying to sync the conditionals though as you mentioned.  If so, maybe just add a comment about that.  I'm pretty sure no one is using Clang 4.x at this point though :).

Reply:
It's strange. I questioned that as well and did some research into how that version got in there. It came in with the initial commit adding clan support to Linux: 8015252: Enable HotSpot build with Clang
https://github.com/battleblow/openjdk-jdk8u/commit/720eac2870b5dd28fece383cd93a10eb3331ae17#diff-ee01ca8848451b0a4ad1995f23a05652aa9eadcd75b9304e7755975ad5048dc0R306
I checked clang releases and there's no 4.2 clang release:
https://releases.llvm.org/
So after scratching my head a bit, I thought perhaps clang was reporting gcc versions numbers back then. Since I wasn't sure I just copied it to match what was there.

I have no strong feelings about it... If you prefer we can remove the 4.2. blocks entirely. Thoughts?

Reply:
Ah found it. Yea clang used to report gcc version numbers with -dumpversion:
https://stackoverflow.com/questions/12893731/why-does-clang-dumpversion-report-4-2-1

Reply:
It appears clang -dumpversion reported gcc versions for comparability until clang 9 was released and then started reporting its own version.

Reply:
Ah, that makes sense.

Reply:
Thanks for the review.